ethereum/.envrc-example

```diff
@@ -9,3 +9,6 @@ export BASIC_CHANNEL_PRINCIPAL=0x0000000000000000000000000000000000000042
 # Incentivized Channel
 # Default fee is 1 SnowDOT (12 decimal places)
 export INCENTIVIZED_CHANNEL_FEE=1000000000000000000
+
+# ParachainClient needs to be constructed with our parachain ID
+export PARACHAIN_ID=1000
```

Comment:
+1 this will get overridden from the Taskfile in snowbridge-infra.

Reply:
Cool, sounds good :+1: